src/move/econia/sources/market.move

```diff
@@ -628,6 +613,34 @@ module econia::market {
         map: Tablist<u64, OrderBook>
     }
 
+    struct MarketEventHandles has key {
+        /// Not emitted at user level when swapper doesn't sign.
+        place_swap_order_events:
+            Tablist<u64, EventHandle<PlaceSwapOrderEvent>>
+    }
+
+    /// Stored under a signing user's account (not market account),
```

Comment:
Shouldn't this be in the `user` module if it's stored under a user's account?

Reply:
It is stored as a resource under the same address as a user's `aptos_framework::account::Account`, which is different from a `econia::user::MarketAccount`: swaps take place outside of an Econia-style `MarketAccount`.

Hence in contrast to an operation through a market account, which requires either a signature or custodian approval, swaps only sometimes require a signature (see https://github.com/econia-labs/econia/pull/321#issuecomment-1624493623)